zone.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 The Inet.Af AUTHORS. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !go1.19
```

Comment:
Why 1.19, rather than "the next release" (so, 1.17-ish) ?

Reply:
@bradfitz suggested 1.19. I'm not sure why.